src/pybind/mgr/dashboard/cherrypy_backports.py

```diff
@@ -94,10 +94,14 @@ def wrapper(self, sock):
                         # Check if it's one of the known errors
                         # Errors that are caught by PyOpenSSL, but thrown by
                         # built-in ssl
-                        _block_errors = ('unknown protocol', 'unknown ca',
-                                         'unknown_ca', 'inappropriate fallback',
-                                         'wrong version number',
+                        _block_errors = ('unknown protocol', 'unknown error',
+                                         'unknown ca', 'unknown_ca',
+                                         'inappropriate fallback', 'https proxy request',
+                                         'wrong version number', 'bad_certificate',
```

Comment:
Where does `bad_certificate` come from? I can't find it in https://github.com/cherrypy/cheroot/blob/master/cheroot/ssl/builtin.py.

Reply:
I added it there, i don't know if its a valid one but I am seeing bad certificate error in downstream. So I thought this will fix that issue too. @votdev

Reply:
@nizamial09 could you reorder these errors to align with `_block_errors` in upstream? if you believe that `'bad_certificate'` should be added as well, could please create a dedicated commit for adding it, and make sure it is reproducible and upstream it to https://github.com/cherrypy/cheroot ?

Reply:
Okay. I'll check if its reproducible in upstream and create a PR.

Reply:
Reproducible in upstream. The error was actually `alert bad certificate`. Updated this PR.

```
ssl.SSLError: [SSL: SSLV3_ALERT_BAD_CERTIFICATE] sslv3 alert bad certificate (_ssl.c:897)
```